instrumentation/github.com/Shopify/sarama/otelsarama/producer.go

```diff
@@ -245,6 +247,31 @@ func WrapAsyncProducer(saramaConfig *sarama.Config, p sarama.AsyncProducer, opts
 	return wrapped
 }
 
+func msgPayloadSize(msg *sarama.ProducerMessage, kafkaVersion sarama.KafkaVersion) int {
+	maximumRecordOverhead := 5*binary.MaxVarintLen32 + binary.MaxVarintLen64 + 1
+	producerMessageOverhead := 26
+	version := 1
+	if kafkaVersion.IsAtLeast(sarama.V0_11_0_0) {
+		version = 2
+	}
+	var size int
+	if version >= 2 {
+		size = maximumRecordOverhead
+		for _, h := range msg.Headers {
+			size += len(h.Key) + len(h.Value) + 2*binary.MaxVarintLen32
+		}
+	} else {
+		size = producerMessageOverhead
+	}
```

Comment:
```suggestion
	size := producerMessageOverhead
	if version >= 2 {
		size = maximumRecordOverhead
		for _, h := range msg.Headers {
			size += len(h.Key) + len(h.Value) + 2*binary.MaxVarintLen32
		}
	}
```

Since `producerMessageOverhead` is a magic number only used here not sure whether it hurts to inline it into the `size` declaration. v0v